tests/disaster-recovery/regional-dr/test_failover.py

```diff
@@ -4,14 +4,15 @@
 from time import sleep
 
 from ocs_ci.framework import config
-from ocs_ci.framework.testlib import rdr_test
+from ocs_ci.framework.testlib import acceptance, tier1
 from ocs_ci.helpers import dr_helpers
 from ocs_ci.ocs.node import wait_for_nodes_status, get_node_objs
 
 logger = logging.getLogger(__name__)
 
 
-@rdr_test
+@acceptance
```

Comment:
the tests will probably fail on a non RDR setup where we will run acceptance and tier1. We should probably add skipif markers

Reply:
ah please ignore.  Just noticed https://github.com/red-hat-storage/ocs-ci/pull/7289/files#diff-5656073f6fc645f6cd203648781fc865dedb793d0dbc437a3c69cf6fd44e29e2R8